flaml/autogen/agentchat/responsive_agent.py

```diff
@@ -73,6 +75,11 @@ def __init__(
                 (3) When "NEVER", the agent will never prompt for human input. Under this mode, the conversation stops
                     when the number of auto reply reaches the max_consecutive_auto_reply or when is_termination_msg is True.
             function_map (dict[str, callable]): Mapping function names (passed to openai) to callable functions.
+            function_call_config (dict or False): config for the function call.
+                - return_limit (Optional, int): default to -1 (no limit), if set, the "on_return_limit" will be triggered.
```

Comment:
what is the int about? token counts?

Reply:
yes. I revised this doc for clear description